base/threadcall.jl

```diff
@@ -75,7 +73,6 @@ function do_threadcall(wrapper::Function, rettype::Type, argtypes::Vector, argva
         y = cconvert(T, x)
         push!(roots, y)
         unsafe_store!(convert(Ptr{T}, ptr), unsafe_convert(T, y))
-        ptr += sizeof(T)
```

Comment:
Without this fix argument passing is broken for me and we should probably add it back.